features/admin.authentication.v1/hooks/use-sign-in.ts

```diff
@@ -556,6 +556,13 @@ const useSignIn = (): UseSignInInterface => {
 
                             sessionStorage.setItem(key, _signOutRedirectURL.href);
                         }
+
+                        if (key.startsWith("config_data-instance_0-CONSOLE")) {
```

Comment:
Lets add an inline comment explaining why we had to manually update the config.

Ref: https://github.com/wso2/identity-apps/blob/e2bccd708dfa6b14e75ee5eb372001699ffb5cd9/features/admin.authentication.v1/hooks/use-sign-in.ts#L538

Reply:
Ack

Reply:
Addressed with 0b74ffa